rc4me/util_test.py

```diff
@@ -18,21 +13,35 @@ def check_repo_files_in_home(repo: Path):
 
 def test_get():
     runner = CliRunner()
-    result = runner.invoke(cli, ["get", "jeffmm/vimrc"])
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
     assert result.exit_code == 0
-    repo = Path("~/.rc4me/jeffmm_vimrc").expanduser()
+    repo = Path("~/.rc4me/mstefferson_rc-demo").expanduser()
     assert repo.exists()
     assert repo.is_dir()
     check_repo_files_in_home(repo)
 
 
 def test_revert():
-    pass
+    runner = CliRunner()
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
+    assert result.exit_code == 0
+    ms_repo = Path("~/.rc4me/mstefferson_rc-demo").expanduser()
+    check_repo_files_in_home(ms_repo)
+    result = runner.invoke(cli, ["get", "jeffmm/vimrc"])
+    assert result.exit_code == 0
+    jm_repo = Path("~/.rc4me/jeffmm_vimrc").expanduser()
+    check_repo_files_in_home(jm_repo)
+    result = runner.invoke(cli, ["revert"])
+    assert result.exit_code == 0
+    check_repo_files_in_home(ms_repo)
 
 
 def test_reset():
-    pass
-
-
-def test_get_local():
-    pass
+    runner = CliRunner()
+    result = runner.invoke(cli, ["get", "mstefferson/rc-demo"])
+    repo = Path("~/.rc4me/mstefferson_rc-demo").expanduser()
+    assert result.exit_code == 0
+    result = runner.invoke(cli, ["reset"])
+    assert result.exit_code == 0
+    repo = Path("~/.rc4me/init").expanduser()
```

Comment:
Should we check the contents of the files?

Reply:
Yes, I'll add issue